app/src/main/res/layout/activity_main.xml

```diff
@@ -24,4 +27,20 @@
             android:layout_height="wrap_content"
             android:text="@string/false_button" />
     </LinearLayout>
+    <LinearLayout
```

Comment:
this is not correctly identated try doing contrl + l in this file and check the results